### PR TITLE
Minutes Technical Lead team Call 2022-12-13

### PR DESCRIPTION
**Pull request by valeriupredoi**
@remi-kazeroni all your for hacking it to pieces, bud :grin:

### CODE REVIEW

File: Tech Lead/Minutes/20221213.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+1. [Organization of the meetings](https://github.com/ESMValGroup/Community/discussions/62#discussioncomment-4148598)
+**Point decisions:**
+
+  V @valeriupredoi proposes we decide who does a number of the following:
+   - V sets up a doodle and sends it around
+   - V talks during the call
+   - Remi @remi-kazeroni **and** V take minutes during the call (items to be decided between Remi and V)
+   - Remi and V collate and PR minutes after the call (alternated)
+   - frequency schedule: once a month unless pressing issues to discuss
```
Comment (remi-kazeroni) [inline Tech Lead/Minutes/20221213.md:28]:
```suggestion
   - frequency schedule: once a month unless pressing issues to discuss
   - V or Remi set up a Discussion page in the [Community repository](https://github.com/ESMValGroup/Community/discussions) to collect issues for the next call
```

File: Tech Lead/Minutes/20221213.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Topics added to the [discussion item page](https://github.com/ESMValGroup/Community/discussions/62)
+
+1. [Organization of the meetings](https://github.com/ESMValGroup/Community/discussions/62#discussioncomment-4148598)
+**Point decisions:**
+
+  V @valeriupredoi proposes we decide who does a number of the following:
+   - V sets up a doodle and sends it around
+   - V talks during the call
+   - Remi @remi-kazeroni **and** V take minutes during the call (items to be decided between Remi and V)
```
Comment (remi-kazeroni) [inline Tech Lead/Minutes/20221213.md:26]:
```suggestion
   - Remi @remi-kazeroni prepares the agenda for the call. The order of the discussion items could be quickly discussed at the beginning of the call.
   - Remi **and** V take minutes during the call (items to be decided between Remi and V)
```

**Review by remi-kazeroni — APPROVED** [review]
Thanks a lot @valeriupredoi for drafting the minutes! I'd have 2 additions regarding the Organization of the meetings. But that's good be be merged otherwise 👍

**Comment by valeriupredoi** [comment]
awesome, thanks a lot for the (nice and easy in-line) comments, bud! :christmas_tree:

**Pull request merged**